ogc-example/src/main/resources/logback.xml

```diff
@@ -7,6 +7,7 @@
 
   <logger name="geotrellis.server.ogc" level="${GT_OGC_LOG_LEVEL:-INFO}"/>
   <logger name="org.http4s.blaze.channel.nio1" level="WARN"/>
+  <logger name="software.amazon.awssdk" level="WARN"/>
```

Comment:
This lib can be a bit chatty, though the warnings can be useful

Reply:
Can we make it also configurable? probably other global vars?